software/apps/audio_module/event_detector/README.md

```diff
@@ -0,0 +1,14 @@
+# Fixed-point ridge tracker for acoustic event detection in C
```

Comment:
I guess this README is just copied from the original repo, and maybe with proper modifications to the Makefile, that's just a submodule or something. However, I think it would be good to have a signpost-port-specific README that

  1. explains a bit more about what the app actually does (e.g. "Detects acoustic events", what "acoustic events" are, etc),

  2. and gives instructions relevant to Signpost (e.g. you can't run `main` with arguments on Signposts).